datafusion/common/src/cast.rs

```diff
@@ -93,14 +131,9 @@ pub fn as_string_view_array(array: &dyn Array) -> Result<&StringViewArray> {
     Ok(downcast_value!(array, StringViewArray))
 }
 
-// Downcast ArrayRef to UInt32Array
-pub fn as_uint32_array(array: &dyn Array) -> Result<&UInt32Array> {
```

Comment:
these were just moved elsewhere in this file I think

Reply:
yes, i moved 2-3 functions to make this file a bit more ordered. maybe i should have abstained from that